Source/interactivity_manager_impl.cpp

```diff
@@ -1167,6 +1187,10 @@ interactivity_manager_impl::on_socket_message_received(
     _In_ const string_t& message
 )
 {
+    // We always return all messages from the service to to the developer, even the ones the SDK will process them.
+    std::shared_ptr<interactive_message_event_args> args = std::shared_ptr<interactive_message_event_args>(new interactive_message_event_args(message));
+    queue_interactive_event_for_client(L"", std::error_code(0, std::generic_category()), interactive_event_type::unknown, args);

```

Comment:
I'd rather this queueing happen in the processing threads, so we keep all logic that pushes messages to clients in the same place. Fine to send it along to them, but this logic should all reside in roughly the same place.